### PR TITLE
Admin Test Create Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Admin Test Create Material

### CODE REVIEW

File: tests/Browser/Pages/Materials/CreateMaterialTest.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                    ->press('Create')
+                    ->assertPathIs('/materials/create')
+                    ->assertSee('The name field is required.')
+                    ->assertSee('The category id field is required.')
+                    ->assertSee('The supplier id field is required.')
+                    ->assertSee('The price field is required.')
+                    ->assertSee('The description field is required.');
+        });
+    }
+
```
Comment (ghost) [inline tests/Browser/Pages/Materials/CreateMaterialTest.php:53]: Please add case create a material with a name already has in the database.

**Review by ghost — COMMENTED** [review]
Please check my comment

File: tests/Browser/Pages/Materials/CreateMaterialTest.php (hunk 1 above)
Comment (vhhieu95) [reply tests/Browser/Pages/Materials/CreateMaterialTest.php:53]: 'The name has already been taken.' cái này có ở phần test testCreateMaterialFailValidation rồi anh.

**Comment by ghost** [comment]
LGTM 💇‍♂️

File: tests/Browser/Pages/Materials/CreateMaterialTest.php (hunk 1 above)
Comment (ghost) [reply tests/Browser/Pages/Materials/CreateMaterialTest.php:53]: OK em, anh thấy rồi

**Pull request merged**